utils/create-atomic-commons/src/console.ts

```diff
@@ -0,0 +1,22 @@
+export const successMessage = (componentName: string) => {
+  console.log(`
+  Project successfully configured
+
+  We suggest that you begin by typing:
+
+  $ cd ${componentName}
+  $ npm install
+  $ npm start
+
+  $ npm start
+    Starts the development server.
+
+  $ npm run build
+    Builds your project in production mode.
+
+  Further reading:
+
+   - TODO: CDX-1403 Add link to documentation in source code and error message
```

Comment:
I would personally remove the TODO from the message, as those have the tendency to slip by in the released product.

We can leave it as a comment in this file though.